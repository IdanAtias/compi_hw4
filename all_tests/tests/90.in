C x = 1 C;
bool b = x > 1 C;
while ( true and b){
	C z = 10 C;
	while( false or b){
		C y;
		input y;
		if ( y >= x and y <= z){
			print "impossible" ;		
		}
	}
}
K y = 100 K;
//num = 10; 
