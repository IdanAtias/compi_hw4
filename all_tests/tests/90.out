-- end scope --
-- end scope --
var:y:3:CELSIUS
-- end scope --
var:z:2:CELSIUS
-- end scope --
var:x:0:CELSIUS
var:b:1:BOOL
var:y:2:KELVIN
